pkg/find/vulnerabilities/trivy/scanner.go

```diff
@@ -152,6 +152,18 @@ func (s *Scanner) PrepareScanJob(_ context.Context, workload kube.Object, spec c
 						},
 					},
 				},
+				{
+					Name: "TRIVY_GITHUB_TOKEN",
```

Comment:
```suggestion
					Name: "GITHUB_TOKEN",
```

I believe it's `GITHUB_TOKEN` as documented in https://github.com/aquasecurity/trivy#github-rate-limiting

Reply:
Fixed.